Yoast/wordpress-seo
# Adds the route for indexing the general indexables

## Context


* We need an endpoint to calculate indexables for general types such as 404, search, homepage and date archives when these aren't calculated yet

## Summary


This PR can be summarized in the following changelog entry:

* Adds an action for calculating the indexables for a 404, search, homepage and the date archive.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Remove all indexables
* Go to Yoast -> Tools and press the index content button.
* Go to your database and see the indexables for search, date archive and 404 pages are calculated. When your frontpage is set as 'posts' (the default options) it will calculate an indexable for the front end as well.

## UI changes

## Documentation

## Quality assurance


Fixes #
